lib/msal-node/src/encode/Base64Decode.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export class Base64Decode {
+    /**
+     * 'utf8': Multibyte encoded Unicode characters. Many web pages and other document formats use UTF-8.
+     * 'base64': Base64 encoding.
+     *
+     * @param base64Str Base64 encoded text
+     */
+    static decode(base64Str: string): string {
+        return Buffer.from(base64Str, 'base64').toString('utf8');
```

Comment:
I think it would be good to have constants for both of these strings.

Reply:
+1

Reply:
Buffer does not accept a 'string', only a set of predetermined strings. 'base64' and 'uint8' being a part of that set.  Hence cannot add them in constants. Will revisit if we can import the buffer constants.